### PR TITLE
Fix norm bug in curvature computation

### PR DESCRIPTION
**Pull request by unalmis**
Looks like the matrix norm was being computed instead of the vector norm for normal vector of a rho surface. Affects all curvature quantities. Might be a good idea to review other places we use norm. `grep -r -F 'np.linalg.norm'`

### CODE REVIEW

File: desc/compute/_geometry.py [hunk 1]
```diff
@@ -233,24 +233,6 @@ def _a_major_over_a_minor(params, transforms, profiles, data, **kwargs):
...
-    description="Unit normal vector to constant rho surface",
-    dim=3,
-    params=[],
-    transforms={},
-    profiles=[],
-    coordinates="rtz",
-    data=["e^rho"],
-)
-def _n_rho(params, transforms, profiles, data, **kwargs):
-    data["n_rho"] = (data["e^rho"].T / jnp.linalg.norm(data["e^rho"])).T
```
Comment (unalmis) [inline desc/compute/_geometry.py:250]: I think this should instead be `jnp.linalg.norm(data["e^rho"]), axis=1).T`.

Comment (dpanici) [reply desc/compute/_geometry.py:250]: I am a little confused, so is `n` wrong or just the `n_rho` quantity?

Comment (unalmis) [reply desc/compute/_geometry.py:250]: it looks like they are supposed to be the same thing? but n was being computed correctly while n_rho wasn't, so i replaced n_rho with n

File: desc/plotting.py [hunk 2]
```diff
@@ -2798,10 +2798,7 @@ def plot_basis(basis, return_data=False, **kwargs):
         fig, ax = plt.subplots(figsize=kwargs.get("figsize", (6, 4)))
 
         f = basis.evaluate(grid.nodes)
-        plot_data = {}
-        plot_data["n"] = basis.modes[:, 2]
-        plot_data["amplitude"] = []
-        plot_data["zeta"] = z
+        plot_data = {"n": basis.modes[:, 2], "amplitude": [], "zeta": z}
```
[the remark this answers is not in the document]
Comment (f0uriest) [reply desc/plotting.py:2801]: no, this `n` is the mode number

**Review by f0uriest — APPROVED** [review]
